ros2/source/images.yaml.em

```diff
@@ -47,3 +42,34 @@ images:
         vcs:
             ros2:
                 repos:
+
+    devel:
```

Comment:
Could we reorder `source, devel` entries in the yaml to `devel, source` to mirror there depency ording?  
This is done with all the other tag configs, e.g. ros-core -> ros-base -> ...